public/components/map_container/map_container.tsx

```diff
@@ -161,6 +161,11 @@ export const MapContainer = ({
     return () => clearInterval(intervalId);
   }, [refreshConfig]);
 
+  // Update data layers when global filter is updated
+  useEffect(() => {
+    renderDataLayers(layers, mapState, services, maplibreRef, timeRange, filters, query);
+  }, [mapState.spatialMetaFilters]);
```

Comment:
Where spatialMetaFilters will be set? I don't see this part in the code

Reply:
Yes. That will be in next PR.